sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py

```diff
@@ -24,11 +24,12 @@ def __init__(
             additionally_allowed_tenants: Optional[List[str]] = None,
             # allow_broker: Optional[bool] = None,
             authority: Optional[str] = None,
-            disable_instance_discovery: Optional[bool] = None,
+            disable_authority_validation_and_instance_discovery: Optional[bool] = None,
```

Comment:
Are we going to add a `:keyword` docstring for this arg for every credential that inherits `MsalCredential`?